src/DataManagement/Saving/SavingCalculator.cs

```diff
@@ -4,113 +4,144 @@
 using ePiggyWeb.DataManagement.Entries;
 using ePiggyWeb.DataManagement.Goals;
 using ePiggyWeb.Utilities;
-using Microsoft.Extensions.Configuration;
 
 namespace ePiggyWeb.DataManagement.Saving
 {
     public class SavingCalculator
     {
         private static decimal RegularSavingValue { get; } = 0.25M;
-        private static decimal MaximalSavingValue { get; } = 0.1M;
-        private static decimal MinimalSavingValue { get; } = 0.5M;
+        private static decimal MaximalSavingValue { get; } = 0.5M;
+        private static decimal MinimalSavingValue { get; } = 0.1M;
+        private List<ISavingSuggestion> EntrySuggestions { get; set; }
+        private List<SavingSuggestionByImportance> SavingSuggestionByImportanceList { get; set; }
+        private IEntryList ExpenseList { get; }
+        private IEntryList IncomeList { get; }
+        public decimal MonthlyIncome { get; set; }
+        private IGoal Goal { get; }
+        private decimal StartingBalance { get; }
+        private SavingType SavingType { get; set; }
+        private int EnumCount { get; }
+        private int MonthsToSave { get; set; }
 
-        public CalculationResults GetSuggestedExpensesOffers(IEntryList entryList, IGoal goal, decimal startingBalance, SavingType savingType, IConfiguration configuration)
+        public SavingCalculator(IEntryList expenseList, IEntryList incomeList, IGoal goal, decimal startingBalance)
         {
-            var entrySuggestions = new List<ISavingSuggestion>();
-            var monthlySuggestions = new List<SavingSuggestionByImportance>();
-            var expensesRandomList = EntryList.RandomList(configuration, EntryType.Expense);
-            var generateRandomData = false;
+            // Reikia gauti average income
+            // Tada reikia, kad kas menesi prisidetu recurring + average per menesi
+            // I menesi sutaupai = averageIncome + savedInAMonth
+            // savedInAMonth = 
 
```

Comment:
Are you going to implement this yourself or this was meant for somebody else?

Reply:
Left this here by mistake